build-locally.sh

```diff
@@ -0,0 +1,154 @@
+#
+# Objectives: Build this repository code locally.
+# 
+#-----------------------------------------------------------------------------------------                   
+
+# Where is this script executing from ?
+BASEDIR=$(dirname "$0");pushd $BASEDIR 2>&1 >> /dev/null ;BASEDIR=$(pwd);popd 2>&1 >> /dev/null
+# echo "Running from directory ${BASEDIR}"
+export ORIGINAL_DIR=$(pwd)
+# cd "${BASEDIR}"
+
+cd "${BASEDIR}/.."
+WORKSPACE_DIR=$(pwd)
+
+
+#-----------------------------------------------------------------------------------------                   
+#
+# Set Colors
+#
+#-----------------------------------------------------------------------------------------                   
+bold=$(tput bold)
+underline=$(tput sgr 0 1)
+reset=$(tput sgr0)
+red=$(tput setaf 1)
+green=$(tput setaf 76)
+white=$(tput setaf 7)
+tan=$(tput setaf 202)
+blue=$(tput setaf 25)
+
+#-----------------------------------------------------------------------------------------                   
+#
+# Headers and Logging
+#
+#-----------------------------------------------------------------------------------------                   
+underline() { printf "${underline}${bold}%s${reset}\n" "$@"
+}
+h1() { printf "\n${underline}${bold}${blue}%s${reset}\n" "$@"
+}
+h2() { printf "\n${underline}${bold}${white}%s${reset}\n" "$@"
+}
+debug() { printf "${white}[.] %s${reset}\n" "$@"
+}
+info()  { printf "${white}[➜] %s${reset}\n" "$@"
+}
+success() { printf "${white}[${green}✔${white}] ${green}%s${reset}\n" "$@"
+}
+error() { printf "${white}[${red}✖${white}] ${red}%s${reset}\n" "$@"
+}
+warn() { printf "${white}[${tan}➜${white}] ${tan}%s${reset}\n" "$@"
+}
+bold() { printf "${bold}%s${reset}\n" "$@"
+}
+note() { printf "\n${underline}${bold}${blue}Note:${reset} ${blue}%s${reset}\n" "$@"
+}
+
+#-----------------------------------------------------------------------------------------                   
+# Functions
+#-----------------------------------------------------------------------------------------                   
+function usage {
+    info "Syntax: build-locally.sh [OPTIONS]"
+    cat << EOF
+Options are:
+-h | --help : Display this help text
+
+Environment Variables:
+SOURCE_MAVEN :
+    Used to indicate where parts of the OBR can be obtained.
+    Optional. Defaults to https://galasadev-cicsk8s.hursley.ibm.com/main/maven/obr/
+     
+LOGS_DIR :
+    Controls where logs are placed. 
+    Optional. Defaults to creating a new temporary folder
+
+GPG_PASSPHRASE :
+    Mandatory.
+    Controls how the obr is signed. Needs to be the alias of the private key of a 
+    public-private gpg pair. eg: For development you could use your signing github
+    passphrase.
+
+EOF
+}
+
+#-----------------------------------------------------------------------------------------                   
+# Process parameters
+#-----------------------------------------------------------------------------------------                   
+exportbuild_type=""
```

Comment:
Needs a space between "export" and "build_type", though this variable isn't being used.